reference/tools.rst

```diff
@@ -283,7 +283,7 @@ See an usage exampel with an environment variable defined while executing conan
 Parameters:
    - **env_key** (Required): environment variable name.
    - **default** (Optional, Defaulted to ``None``): default value to return if not defined or cast value against.
-   - **environment** (Optional, Defaulted to ``os.environ``): environment dictionary to look for.
+   - **environment** (Optional, Defaulted to ``None``): os.environ if None or environment dictionary to look for.
```

Comment:
I would say ``os.envion`` and ``None`` as it is python code

Reply:
@danimtb I don't get it, please rephrase it with your suggestion

Reply:
It is Ok, just this details:
```
- **environment** (Optional, Defaulted to ``None``): ``os.environ`` if ``None`` or environment dictionary to look for.